packages/cli-lib/lang/en.lyaml

```diff
@@ -1,6 +1,8 @@
 en:
   cli:
     commands:
+      generalErrors:
+        srcIsProject: "\"{{ src }}\" is a project folder. Please use a command with the \"hs project\" prefix."
```

Comment:
Copy suggestion: `"\"{{ src }}\" is a project folder. Did you mean \"hs project upload\"?"`

The "upload" portion would vary depending on which command the user ran